packages/webpack/test/e2e-lockdown-inline.spec.js

```diff
@@ -4,7 +4,7 @@ const { makeConfig } = require('./fixtures/main/webpack.config.js')
 
 test.before(async (t) => {
   const webpackConfig = makeConfig({
-    inlineLockdown: ['app.js'],
+    inlineLockdown: /app/,
```

Comment:
idk if this is a snapshot test, but it seems like there should be such a thing

Reply:
this test runs the bundle and checks if lockdown was applied. 
a snapshot test would have to snapshot the bundle source and it would be flaky.